docs/rc.xml.all

```diff
@@ -89,6 +89,17 @@
     </names>
   </desktops>
 
+  <!--
+    You can reserve portions of your screens where windows will not cover when
+    they are maximized/tiled/etc or when they are initially placed. Clients
```

Comment:
You can reserve portions of your screens which will not be used by windows in maximized or tiled states.

Reply:
Looking at the man page, I opted for "Specify the number of pixels to reserve at the edges of an output. New, maximized and tiled windows will not be placed in these areas." I'll patch it with something along those lines.

Reply:
sounds good.

Reply:
Done